Watchman.Discord/Areas/Administration/Controllers/AdministrationController.cs

```diff
@@ -92,51 +92,52 @@ public async Task ReadUserMessages(MessagesCommand command, Contexts contexts)
         }
 
         [AdminCommand]
-        public async Task SetRoleAsSafe(SetRoleCommand command, Contexts contexts)
+        public Task SetRoleAsSafe(SetRoleCommand command, Contexts contexts)
         {
             if (!command.Safe && !command.Unsafe)
             {
                 throw new NotEnoughArgumentsException();
             }
```

Comment:
```suggestion
            return command.Safe || command.Unsafe
                ? this._rolesService.SetRolesAsSafe(contexts, command.Roles, setAsSafe: command.Safe)
                : throw new NotEnoughArgumentsException();
```
niech tak wygląda ciało tej metody (SetRoleAsSafe)

Reply:
Jak dla mnie tak jak jest obecnie lepiej to wygląda